### PR TITLE
Merging Requirements and Getting Started page

### PR DESCRIPTION
**Pull request by OfficePop**
Merging requirements.md content into getting-started per [kubeflow/website issue 845](https://github.com/kubeflow/website/issues/845). Changes are as follows:

1. Contents from requirements.md were added to getting-started.md. Slight changes to sub-heading information.
2. Requirements.md was deleted.
3. A re-direct from requirements.md to getting-started.md was added to _references.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/923)

### CODE REVIEW

File: content/docs/started/getting-started.md [hunk 1]
```diff
@@ -1,9 +1,32 @@
 +++
 title = "Getting Started with Kubeflow"
-description = "Get your machine-learning workflow up and running on Kubeflow"
+description = "Overview"
 weight = 1
 +++
 
+The guides in this section give detailed information about using Kubeflow and
```
[the remark this answers is not in the document]
Comment (OfficePop) [reply content/docs/started/getting-started.md]: Thanks for the suggestion! I've updated with a commit covering this comment and the next two as well.

**Comment by sarahmaddox** [comment]
Thanks @OfficePop for this update and @joeliedtke for the review!

/lgtm
/approve

**Pull request merged**